src/server/spectrum_files.R

```diff
@@ -1,75 +1,72 @@
-library(shiny)
-library(purrr)
-library(glue)
 library(first90)
 
 spectrumFiles <- function(input, output, state) {
     state$country <- NULL
-    state$anyDataSets <- reactive({ length(state$dataSets) > 0 })
-    state$combinedData <- reactive({
+    state$anyDataSets <- shiny::reactive({ length(state$dataSets) > 0 })
+    state$combinedData <-shiny::reactive({
         if (state$anyDataSets()) {
             # TODO use all files, not just first one
             state$dataSets[[1]]$data
         }
     })
 
-    observeEvent(input$spectrumFile, {
+    shiny::observeEvent(input$spectrumFile, {
         inFile <- input$spectrumFile
         if (!is.null(inFile)) {
             dataSet = list(
                 name = inFile$name,
-                data = prepare_inputs(inFile$datapath)
+                data = first90::prepare_inputs(inFile$datapath)
```

Comment:
When I don't load the `first90` package this line errors with `Warning: Error in prepare_directincid: could not find function "prepare_directincid"`. I assume because `first90` is itself loading another package which includes that function? But I can't work out which one.

Reply:
Ok, loading `eppasm` seems to be enough for this to work, but puzzled by the fact that I can't find `prepare_directincid` in the eppasm repo :woman_shrugging: